crates/ruff_linter/src/checkers/ast/analyze/definitions.rs

```diff
@@ -149,23 +150,22 @@ pub(crate) fn definitions(checker: &mut Checker) {
             if checker.enabled(Rule::DocstringInStub) {
                 flake8_pyi::rules::docstring_in_stubs(checker, docstring);
             }
+            if checker.enabled(Rule::OldStylePositionalOnlyArg) {
+                flake8_pyi::rules::old_style_positional_only_arg(checker, definition);
+            }
```

Comment:
We should make sure only to run this rule if the target version is set to Python 3.8 or greater (Ruff still supports Python 3.7), or we'll be recommending invalid syntax for Python 3.7 users